go/vt/vtgate/planbuilder/testdata/reference_cases.json

```diff
@@ -700,19 +678,19 @@
       "QueryType": "SELECT",
       "Original": "select * from user.ref_with_source ref, `user`.`user` u where ref.id = u.ref_id and u.id = 2",
       "Instructions": {
-        "FieldQuery": "select * from ref_with_source as ref, `user` as u where 1 != 1",
```

Comment:
Trying to understand what happened here. Why did these fields change places?

Reply:
I have no idea either, but I'll look into it.